package.json

```diff
@@ -8,47 +10,109 @@
     "type": "git",
     "url": "git@github.com:videojs/videojs-contrib-hls.git"
   },
-  "license": "Apache-2.0",
   "scripts": {
-    "test": "grunt test",
-    "prepublish": "if [ -z \"$TRAVIS\" ]; then grunt; fi"
+    "prebuild": "npm run clean",
+    "build": "npm-run-all -p build:*",
+    "build:js": "npm-run-all build:js:babel build:js:browserify build:js:bannerize build:js:uglify",
+    "build:js:babel": "babel src -d es5",
+    "build:js:bannerize": "bannerize dist/videojs-contrib-hls.js --banner=scripts/banner.ejs",
+    "build:js:browserify": "browserify . -s src/plugin.js -o dist/videojs-contrib-hls.js",
+    "build:js:uglify": "uglifyjs dist/videojs-contrib-hls.js --comments --mangle --compress -o dist/videojs-contrib-hls.min.js",
+    "build:test": "node scripts/build-test.js",
+    "clean": "node -e \"var s=require('shelljs'),d=['dist','dist-test','es5'];s.rm('-rf',d);s.mkdir('-p',d);\"",
+    "docs": "npm-run-all docs:*",
+    "docs:api": "jsdoc src -r -d docs/api",
+    "docs:toc": "doctoc README.md",
+    "lint": "vjsstandard :",
+    "prestart": "npm-run-all docs build",
+    "start": "npm-run-all -p start:* watch:*",
+    "start:serve": "babel-node scripts/server.js",
+    "pretest": "npm-run-all lint build",
```

Comment:
If you remove `lint` from this *for now*, it won't fail linting. We can then have a separate PRs for the browserification and es6ification and then we can add it back in then.